README.md

```diff
@@ -22,7 +22,56 @@ This token will need:
 * Read/Write access to Linodes (to attach and detach volumes)
 * A sufficient "Expiry" to allow for continued use of your volumes
 
-### Create a Kubernetes secret
+### Deploying the CSI Driver
+
+There are two ways of deploying CSI. The recommended way is to use Helm and second way is to use manually set secrets on the kubernetes cluster then use kubectl to deploy using the given [`yaml` file](https://raw.githubusercontent.com/linode/linode-blockstorage-csi-driver/master/pkg/linode-bs/deploy/releases/linode-blockstorage-csi-driver.yaml). 
```

Comment:
I'd recommend moving this line up to line 9 under Deployment and removing line 25